src/main/java/org/jabref/gui/IconTheme.java

```diff
@@ -48,16 +48,10 @@
     // Christmas edition
     //public static final Color DEFAULT_COLOR = new Color(0x155115);
     //public static final Color DEFAULT_DISABLED_COLOR = new Color(0x990000);
-    private static Font FONT_16;
```

Comment:
This should not have been removed as it came from the master branch. - I think, I got it back by a subsequent merge.